src/libs/actions/Report.js

```diff
@@ -422,7 +422,7 @@ function navigateToAndOpenReport(userLogins) {
  *
  * @param {String} reportID
  */
-function reconnect(reportID) {
+function reconnectToReport(reportID) {
```

Comment:
I renamed the function because I thought this was clearer and makes the function name the same as the API call (similar to `openReport`)

Reply:
I don't feel super strongly about it. Looking around the codebase, it seems that we often use the same function name as the API call, but not all the time. @aldo-expensify what do you think?

Reply:
I would prefer to revert because it is not clear that this is more consistent considering the rest of the code:

There is another `reconnect` in `Pusher` (not `Pusher.reconnectToPusher`): https://github.com/Expensify/App/blob/93e1d3b84726a66607715872c0a63531d0e82b16/src/libs/Pusher/pusher.js#L363-L372

It appears to me that we are not being consistent with the naming of the functions in the [src/libs/actions/Report.js](https://github.com/Expensify/App/blob/93e1d3b84726a66607715872c0a63531d0e82b16/src/libs/actions/Report.js), for example:

- `saveReportComment`: has report on the name
- `addAttachment`: why this is not `addReportAttachment`
- `addComment`: `why this is not `addReportComment`

I don't think we should start renaming if we have not agreed on a direction. I lean toward removing the "Report" word from the function names because we usually call this functions with `Report.something` so `Report.` is already there, but that is just my personal opinion on it :)

Reply:
>I don't think we should start renaming if we have not agreed on a direction.

This makes a lot of sense. Thanks for the thoughts both of you. I'll go ahead and change the name back.